README.md

```diff
@@ -31,9 +31,16 @@ Please check [VDVS Installation User Guide](http://vmware.github.io/vsphere-stor
 
 ## Supported Platforms
 
-**ESXi:** 6.0 and above<br />
-**Docker (Linux):** 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)<br />
-**Docker (Windows):** 1.13/17.03 and above (Windows containers mode only)
+**ESXi:** 6.0U2 and above<br />
+**Docker (Linux):** 17.06.1 and above to use managed plugin (v17.06.1 is having an important fix for data loss issue)<br />
```

Comment:
Please remove these details - v17.06.1 is having an important fix for data loss issue

Reply:
Done!